torch/fx/experimental/symbolic_shapes.py

```diff
@@ -594,7 +595,7 @@ class SymNode:
     This is a type erased SymInt/SymFloat which we use to do actual operations.
     End users don't touch this.  Magic methods are NOT defined on this object.
     """
-    def __init__(self, expr, shape_env, pytype, hint: Optional[Union[int, float]], constant=None):
+    def __init__(self, expr, shape_env, pytype, hint: Optional[Union[int, float]], constant=None, fxnode=None):
```

Comment:
nittiest of nits, but the spelling fxnode rankles me cuz it feels like it should be `fx_node` lol